src/app/models/book-borrow.ts

```diff
@@ -0,0 +1,15 @@
+import { returnBookOption } from '../return-book-type.enum';
+
+export class BookBorrow {
+  public dateBorrowBook: Date;
```

Comment:
one more rename... Sorry ;) I think 'borrowBookDate' feels more natural. But that's just detail, can be left as it is.

Reply:
trueeeee! Thank you <3